### ISSUE TITLE
Process Types and Transfer Types in circular economy

### ISSUE THREAD

**Issue by fosterlynn** [valueflows/forum.valueflo.ws#43]
Interesting list from @pmackay around what they have developed as a list of process types and transfer types or exchange types.  And a way to include the concept of good > bad in terms of effect in the circular economy.

From Gitter:
pmackay
12:10
FYI, here's the list of exchange types and processes we've worked on for the circular economy (no doubt there are more):
Renting
Borrowing
Lending
Reuse
Repairing
Donating
Selling
Giving / Gifting
Swapping
Repurposing
Refurbish
Upcycling
Recycling
Closed loop recycling
Downcycling
Composting
Home composting
Home digestion
Anerobic digestion
Disposal
Landfill
Pyrolisis
Gasification
fosterlynn
12:14
@pmackay cool, studying.

```
Do you think its possible to have a common set of Types defined? 
```

Sometimes it is and sometimes is isn't, because it is important for emerging groups to work through how they want to define the next economy, there are lots of experiments in this transitional stage.
(We had a lot of discussion on this in Agent definition, because @elf-pavlik coming out of LOD perspective wanted to try to define the Agent Relationship Types. Which I get. But we just aren't ready to do that.)
@pmackay looks like your list is partially specific to your domain, and partially pretty general. Still studying....
bhaugen
12:28
@pmackay nice list, though, I like it!
pmackay
12:29
basically all methods of what can done with a thing, going from good to worse in terms of levels of circular economy. Reduce/reuse at top, dispose at bottom, etc.
bhaugen
12:29
@/all - first draft of https://github.com/valueflows/valueflows/wiki/Provenance
more to come, later today maybe
fosterlynn
12:30
thanks @bhaugen ; @pmackay i didn't mean to be critical! i like your list too!
bhaugen
12:30
good-to-worse is an interesting relationship among process types
I wonder if that will have analogs in other domains?
pmackay
12:31
of course its only a good > bad relationship in context of circular economy and waste hierarchy

**Comment by almereyda** [comment]
We have moved the ValueFlows organization from GitHub to https://lab.allmende.io/valueflows.

This issue has been closed here, and all further discussion on this issue can be done at

> https://lab.allmende.io/valueflows/forum-valueflo-ws/-/issues/43.

If you have not done so, you are very welcome to register at https://lab.allmende.io and join the ValueFlows organization there.
